src/pages/Dashboard.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import configureStore from 'redux-mock-store';
+
+import { initialState } from 'reducers/dashboard';
+import ConnectedDashboard from './Dashboard';
+import { withMemoryRouter } from 'utils';
+
+describe('<Dashboard />', () => {
+  const mockStore = configureStore();
+  const store = mockStore({ dashboard: initialState });
```

Comment:
To fill the snapshot with actual data, pass a modified initialState into mockStore. Otherwise the snapshot is just a test that there is no rendering error, not a test of whether or not the component can display correctly with data.

Reply:
initialState loads the dashboard homepage, I am currently testing data in Main.js

Reply:
Gotcha, makes sense.